Extensions/BBText/bbtextruntimeobject-pixi-renderer.js

```diff
@@ -0,0 +1,118 @@
+/**
+ * The PIXI.js renderer for the BBCode Text runtime object.
+ *
+ * @class BBTextRuntimeObjectPixiRenderer
+ * @constructor
+ * @param {gdjs.BBTextRuntimeObject} runtimeObject The object to render
+ * @param {gdjs.RuntimeScene} runtimeScene The gdjs.RuntimeScene in which the object is
+ */
+gdjs.BBTextRuntimeObjectPixiRenderer = function(runtimeObject, runtimeScene) {
+  this._object = runtimeObject;
+
+  // Load (or reset) the text
+  if (this._pixiObject === undefined) {
+    this._pixiObject = new MultiStyleText(runtimeObject._text, {
+      default: {
+        fontFamily: runtimeObject._family,
+        fontSize: `${runtimeObject._size}px`,
+        fill: runtimeObject._color,
+        tagStyle: ['[', ']'],
+        wordWrap: true,
+        wordWrapWidth: runtimeObject._wrappingWidth,
+        align: runtimeObject._align,
+      },
+    });
+
+    this._object.hidden = !runtimeObject._visible;
+  } else {
+    console.log('runtime obj', runtimeObject._wrappingWidth, this._pixiObject);
+    // this._pixiObject._texture.baseTexture.source.currentTime = 0;
```

Comment:
In case your object was cached (meaning that a BBText object was created, destroyed, but kept by the game engine in a cache, then revived to serve for a new object that was created), `this._pixiObject` will be still alive (i.e: `this._pixiObject` won't be undefined) and containing the options of the previous object.
Make sure in this `else` to reset everything to the proper values according to `this._object`.

If something will be updated anyway by the end of the constructor:
```
  this.updatePosition();
  this.updateAngle();
  this.updateOpacity();
  this.updateVisible();
```

then add a comment here telling: "No setting X/Y/Z as it will be updated below". But from what I see you still need to update most things (align, wordWrapWidth, fill, fontSize, fontFamily).

Reply:
(FYI, this technic is called a "pool" of object. Might be not used for this kind of objects, but for objects that are rapidly being created and deleted (like bullets), having the game engine doing "recycling" of objects can provide a performance boost. The downside is using a bit more memory and having to be more careful about re-initialising everything like here.)

Reply:
Interesting :) sounds like a great optimisation trick. Thank you for explaining this. Updated the pr now to incorporate resetting

Reply:
I need to add updateText to there too now, since I split it out into its own function